eo-maven-plugin/src/main/java/org/eolang/maven/footprint/Footprint.java

```diff
@@ -29,7 +29,8 @@
 import org.cactoos.Scalar;
 
 /**
- * Program footprint of EO compilation process.
+ * Program footprint of EO compilation process,
+ * convenient eo program management mechanism.
```

Comment:
@levBagryansky It's still not so clear as for me. Maybe we can add more text here like:

>   Footprint refers to a trace of EO program on file system which is quite fortunate analogy with natural meaning of the word.

> The idea of Footprint is to provide abstraction for EO program files content. 

The general idea of that PR is to add one or two sentences that clearly will explain what is `Footprint` and what problem it solves.

Reply:
Or even better - we can provide examples.

Reply:
> Or even better - we can provide examples.
Won't the examples be superfluous? The tests already illustrate the capabilities of the class quite well as for me.

Reply:
@levBagryansky I don't think it will be superfluous. Moreover, you can use simple natural words instead of code. It could explain more and in a single place.